src/helpers.js

```diff
@@ -459,7 +493,57 @@ async function promptInput(message) {
 	return selected.item;
 }
 
+/**
+ *loads the pupa module dynamically and then interpolates the raw data from mesh file with object data
+ * @param {data}
+ * @param {obj}
+ * @returns {object} having interpolationStatus, missingKeys and interpolatedMesh
+ */
+
+async function interpolateMesh(data, obj) {
```

Comment:
Any reason why you didn't put this in `utils.js` with all of the other env/interpolation related functions?